.storybook/preview.js

```diff
@@ -107,4 +107,4 @@ const withAlertContext = (Story, context) => {
     );
 }
 
-export const decorators = [withDarkMode, withAlertContext];
+export const decorators = [withDarkMode];
```

Comment:
Removed withAlertContext decorator as it was re-rendering with each change in value. Needs to be fixed somehow. Changing to {Story(context)} didn't work as it worked with withDarkMode.

Reply:
We can't just remove this, if it was used somewhere else
We will have to defined provider for such stories

Reply:
Yess

Reply:
I've moved it inside AlertContainer stories, and it seems to work.
Could you guys check it out?